### DISCUSSION TITLE
refresh oidc token in login flow

### DISCUSSION THREAD

**Discussion by david972** [ory/kratos#1911] [Ideas]
Hi,

Currently Kratos record the token during the registration and the setting.

If the access_token recorded has expired in the meanwhile or tokens has been revoked it would be useful to update them.
I know in case of expiration we can use the refresh token but not in case of revocation.

Do you see a security concern to update the tokens on login ?
If not I can do a PR

**Comment by aeneasr** [comment]
Makes sense, PRs welcomed!
